components/button/button-icon.js

```diff
@@ -155,6 +156,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 	}
 
 	render() {
+		const icon = this.icon ? html`<d2l-icon icon="${this.icon}" class="d2l-button-icon"></d2l-icon>` : html`<slot></slot>`;
```

Comment:
Wondering if we want to reserve the default slot for future and name this one `<slot name="icon">`?

Reply:
Ah yup good call